src/entity_data.zig

```diff
@@ -173,16 +173,16 @@ fn BlockEntityDataStorage(comptime side: enum {client, server}, T: type) type {
 
 pub const EntityDataClasses = struct {
 	pub const Chest = struct {
-		const StorageServer = BlockEntityDataStorage(
+		pub const StorageServer = BlockEntityDataStorage(
 			.server,
 			struct {
-				contents: u64,
+				inventory: ?main.items.Inventory,
 			},
 		);
-		const StorageClient = BlockEntityDataStorage(
+		pub const StorageClient = BlockEntityDataStorage(
 			.client,
 			struct {
-				contents: u64,
+				inventory: ?main.items.Inventory,
```

Comment:
As discussed in #1224, the client side does not need to store the inventory for all chest blocks.